partcad/src/partcad/sketch_factory_enrich.py

```
@@ -19,45 +19,62 @@
             # Get the config of the sketch the 'enrich' points to
-            orig_source_project = source_project
-            if self.source_project_name == source_project.name:
-                augmented_config = source_project.get_sketch_config(self.source_sketch_name)
+            if self.source_project_name == self.source_project.name:
+                augmented_config = self.source_project.get_sketch_config(self.source_sketch_name)
             else:
-                source_project = ctx.get_project(self.source_project_name)
-                augmented_config = source_project.get_sketch_config(self.source_sketch_name)
+                self.source_project = self.ctx.get_project(self.source_project_name)
+                if self.source_project is None:
+                    pc_logging.debug("Available projects: %s" % str(sorted(list(self.ctx.projects.keys()))))
```

Ruff (0.8.2)

Use format specifiers instead of percent format

Replace with format specifiers

(UP031)